### DISCUSSION TITLE
Rosetta stonks

### DISCUSSION THREAD

**Discussion by Ayb524** [m1dugh/RosettaStonks#102] [General]
Hello can you please share explaining video for how to use Rosetta stonks with us

**Comment by np4abdou** [comment]
@m1dugh please update so it would work on fleuncy builder it work sometimes not

**Reply by Ayb524** [reply]
Bro can you tech me how it works plz

**Reply by np4abdou** [reply]
I made a tutorial in my tiktok : @this_is_abdou_

**Reply by messaoud-laaouaj1** [reply]
> I made a tutorial in my tiktok : @this_is_abdou_

The method does not work on fluency 
I waited 3 days for the time and it has not increased yet  plls hlp me

**Reply by voidd159** [reply]
did it work for u yet? cz i used it abt a week ago n it worked

**Reply by nader234623** [reply]
it did work but after a long time so it's usefull

**Reply by chihabend** [reply]
hello bro i think roseta stone updated web site the time don't added please if you can fix the probleme i tried but i can't

**Comment by AlexandreGirold** [comment]
> Hello can you please share explaining video for how to use Rosetta stonks with us

Maybe soon, for now refer to the README and SECURITY.md

**Reply by np4abdou** [reply]
@AlexandreGirold why time only can be added in foundations and not fleuncy builder please update the extension 🙏

**Reply by chihabend** [reply]
hello bro i think roseta stone updated web site the time don't added please if you can fix the probleme i tried but i can't

**Comment by T1ttySl4yer** [comment]
brothers why is it not working with fluency builder?

**Reply by drewcanflyy** [reply]
bro there's no script for fluency builder ?

**Comment by Isernon** [comment]
Hello @m1dugh , could you try to fix the code please. it's been 3 days and the hours weren't added on fluency builder.

**Reply by messaoud-laaouaj1** [reply]
yes i have the same prblm beo

**Reply by chihabend** [reply]
hello bro i think roseta stone updated web site the time don't added please if you can fix the probleme i tried but i can't

**Comment by xyz6269** [comment]
hi can you document exactly how this extension adds time, it clearly intercepts requests and changes them yes, but won't checking the timestamps reveal that the user is cheating?

**Reply by Ali-Chellat** [reply]
Ig that's private and schools or universities can't detect it ,the report shows how much you have done in total, soo as long as you don't add more than 24hours a day which is impossible, or a big amount like 18 hours which no one will do it manually, it's fine. This remains my personal analysis it could be wrong.

**Reply by Shadows696** [reply]
@Ali-Chellat did it work for you?

**Reply by xyz6269** [reply]
@Ali-Chellat hmmmm the thing is if you remember on this repo there was an issue that the owner deleted now about Moroccan students getting caught cheating and the list contains more than 30 people man, so I'm here wondering if 30 people were stupid enough to add a huge amount in 1 day. So are you sure that universities can't see the timestamps of when you started and finished the activities/lessons, I'm not trying to get fired from my masters program lmao

**Reply by Ali-Chellat** [reply]
> @Ali-Chellat did it work for you?

It worked just fine a week ago, but now just like everyone else it's not working

**Reply by Ali-Chellat** [reply]
> @Ali-Chellat hmmmm the thing is if you remember on this repo there was an issue that the owner deleted now about Moroccan students getting caught cheating and the list contains more than 30 people man, so I'm here wondering if 30 people were stupid enough to add a huge amount in 1 day. So are you sure that universities can't see the timestamps of when you started and finished the activities/lessons, I'm not trying to get fired from my masters program lmao

Aahh yeah it was a school in casablanca and ig ENS Marrackech, the thing is I personally know many people that have now 60 hours done or 100 hours and some even more, soo such cases doesn't have any excuses, like you studied 80 hours on the app? that's impossible, soo yes 30 people is a reasonable amount of people that could be caught, But as I stated before I don't have any official informations about this, it's just my own reasoning based on what I saw and what I know.

**Reply by chihabend** [reply]
hello bro i think roseta stone updated web site the time don't added please if you can fix the probleme i tried but i can't

**Comment by grlod** [comment]
i add time in fluency bulder with 2 methodes and its been a week for the first methode and 3 or 4 days for the 2 methode but no results what should i do please im confused now please answear me

**Reply by grlod** [reply]
@m1dugh

**Reply by voidd159** [reply]
lets hope he updates it asap

**Reply by omarberrada007** [reply]
@m1dugh can u help me with a probleme i have now i mistakenly added 15h in english and 15 hours in french can u erase the added 30h pleasee i beg j

**Reply by AlexandreGirold** [reply]
> @m1dugh can u help me with a probleme i have now i mistakenly added 15h in english and 15 hours in french can u erase the added 30h pleasee i beg j

You can just reset your account directly in Rosetta Stone

**Reply by Twilight392** [reply]
> You can just reset your account directly in Rosetta Stone

i dont think you can

**Reply by omarberrada007** [reply]
Does it actually work ?

**Reply by m1dugh** [reply]
I tried it on my own multiple times and it worked every time. Have you read entirely the readme file and followed the instructions ?

**Reply by MarkvsLoopvs** [reply]
> I tried it on my own multiple times and it worked every time. Have you read entirely the readme file and followed the instructions ?

I am almost certain it doesn't work on fluency builder, followed the readme to a t.

**Reply by Isernon** [reply]
> I tried it on my own multiple times and it worked every time. Have you read entirely the readme file and followed the instructions ?

It doesn't work on fluency builder. Are you sure it works on FLUENCY BUILDER?

**Reply by m1dugh** [reply]
I am sure it works on fluency builder. Did you finish the lesson you used to add time ?

**Reply by MarkvsLoopvs** [reply]
> I am sure it works on fluency builder. Did you finish the lesson you used to add time ?

Yes, I did. Are we talking about the same thing when we say "lesson"? Do you mean an individual module of a course (referred to as "lesson" on the platform) or do you maybe mean a whole course?

**Reply by voidd159** [reply]
> I am sure it works on fluency builder. Did you finish the lesson you used to add time ?

yea, finished yet still not workin

**Reply by chihabend** [reply]
hello bro i think roseta stone updated web site the time don't added please if you can fix the probleme i tried but i can't

**Comment by Badreddine2005** [comment]
any news guys? doest it work ?

**Comment by T1ttySl4yer** [comment]
Hello @m1dugh , I followed the exact instructions in readme and rosetta stonks is successfully activated, but when I added few hours yesterday ,nothing has changed. (Fluency Builder).

**Reply by m1dugh** [reply]
Did you finish the lesson you added hours on ?

**Reply by T1ttySl4yer** [reply]
Honestly no, I only finished one task so I can add the required hours, do I have to finish the whole lesson, or just few tasks?

**Reply by m1dugh** [reply]
You have to finish the whole lesson.

**Reply by m1dugh** [reply]
It is written in the readme

**Reply by Badreddine2005** [reply]
i've tried it and no hours had appeared

**Reply by Ali-Chellat** [reply]
> You have to finish the whole lesson.

This is new right, because a week ago or like 10 days it wasworking without even completing the lesson

**Reply by chihabend** [reply]
hello bro i think roseta stone updated web site the time don't added please if you can fix the probleme i tried but i can't

**Comment by Isernon** [comment]
@m1dugh I just completed a whole lesson on fluency builder to see if it will add time but it didn't work, can you please fix it ?

**Reply by voidd159** [reply]
it acc works u have to wait for a day nd it will show up it worked for me tho

**Reply by T1ttySl4yer** [reply]
How did u manage to make it work , did u have to complete the whole lesson? I did but nothing has changed.
It works well for A1 and A2 levels but not for the other levels

**Reply by wsmusashi** [reply]
> it acc works u have to wait for a day nd it will show up it worked for me tho

tell us how.

**Reply by Isernon** [reply]
> it acc works u have to wait for a day nd it will show up it worked for me tho

Unfortunately, it didn't work. I just checked now (after 24h). Can you please help me since it did work for you on fluency builder because I did what README says step by step and nothing changed it didn't work before and it isn't working now.

**Reply by voidd159** [reply]
> How did u manage to make it work , did u have to complete the whole lesson? I did but nothing has changed. It works well for A1 and A2 levels but not for the other levels

i just did it for a friend of mine who is a b1 yesterday all i did is finish the lesson and make sure to add time after some questions nd after finishin i just click on next lesson and exit lesson thts it and just did wait for the next day

**Reply by Ali-Chellat** [reply]
> > How did u manage to make it work , did u have to complete the whole lesson? I did but nothing has changed. It works well for A1 and A2 levels but not for the other levels
> 
> i just did it for a friend of mine who is a b1 yesterday all i did is finish the lesson and make sure to add time after some questions nd after finishin i just click on next lesson and exit lesson thts it and just did wait for the next day

Oohh man can u help me, I really need to get this to work for my sister and she only has a week left

**Reply by chihabend** [reply]
hello bro i think roseta stone updated web site the time don't added please if you can fix the probleme i tried but i can't

**Comment by chihabend** [comment]
hello bro i think roseta stone updated web site the time don't added please if you can fix the probleme i tried but i can't

**Reply by m1dugh** [reply]
Hello,

It takes up to 3 days to see the added time. Have you waited ?

**Reply by chihabend** [reply]
Yes bro, I waited, but it doesn't work. Can you check if any changes were applied to the website? If you need a Rosetta Stone account, I can give it to you. 
tnx

**Reply by chihabend** [reply]
hi bro any news?

**Reply by Badreddine2005** [reply]
hello, it's been 3 days, and the hours didn't appear yet, any news about this?
